<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>968</width>
    <height>559</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="AddUser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>AddUser</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>941</width>
      <height>201</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
    </property>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>30</number>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>130</number>
    </attribute>
    <column>
     <property name="text">
      <string>Имя</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Фамилия</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Отчество</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>адрес</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>дата рождения</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>почта</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>номера телефонов</string>
     </property>
    </column>
   </widget>
   <widget class="QLineEdit" name="Name">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="LastName">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>220</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="MiddleName">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>220</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="adress">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>220</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="birthdate">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>220</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="phoneNumbers">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>220</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="mail">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>220</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="idOfDelete">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>290</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="DeleteUser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>DeleteUser(id)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="filepath">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>320</y>
      <width>291</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="SaveUser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Users(flePath)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LoadUsers">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Users(flePath)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="filepathtoload">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>350</y>
      <width>291</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="findByRowButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>231</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>FindByRow(Name, Last_Name or Email)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="findByRow">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>380</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="sortFieldLineButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Sort</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="sortFieldLineEdit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>410</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>968</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
